rails/spec/requests/api/public_stories_spec.rb

```diff
@@ -180,5 +215,16 @@
       expect(json_response["stories"].count).to eq(1)
       expect(json_meta["hasNextPage"]).to be false
     end
+
+    it "orphaned stories are excluded" do
+       # ensures that story_with_no_speaker does not show up in list:
+      speaker_being_deleted.destroy
+      # ensures that story_with_no_place does not show up in list:
+      place_being_deleted.destroy
+
+      get "/api/communities/orphaned_community/stories"
+      expect(json_response["total"]).to eq(0)
```

Comment:
The Stories API does not return `total` under `json_response` anymore, it's available under `json_meta`:

```suggestion
      expect(json_meta["total"]).to eq(0)
```